docs/website/content/integrations/regocpp.md

```diff
@@ -0,0 +1,13 @@
+---
+title: regocpp
+software:
+- regopy
+- regorust
+inventors:
+- Microsoft
```

Comment:
```suggestion
- microsoft
```

Reply:
We have an exiting org for MSFT, but it'll need to be lower cased to match.

It'll also appear on this page: https://www.openpolicyagent.org/organizations/microsoft/